betterproto/tests/inputs/to_dict_with_missing_enum/test_to_dict_with_missing_enum.py

```diff
@@ -0,0 +1,16 @@
+from betterproto.tests.output_betterproto.to_dict_with_missing_enum.to_dict_with_missing_enum import (
+    Test,
+    TestMyEnum,
+)
+
+
+def test_message_attributes():
```

Comment:
```suggestion
@pytest.mark.xfail(“#93”)
def test_message_attributes():
```

Reply:
this suggested change (combined with the one below) is to make the tests pass.
sorry, i had not realized this was also still necessary for a merge